library/src/main/res/values/strings.xml

```diff
@@ -47,8 +49,13 @@
     <string name="chucker_share_throwable_content"><![CDATA[Date: %1$s\nException: %2$s\nTag: %3$s\nMessage: %4$s\n\n%5$s]]></string>
     <string name="chucker_clear_http_confirmation">Do you want to clear complete network calls history?</string>
     <string name="chucker_clear_throwable_confirmation">Do you want to clear complete throwables history?</string>
+    <string name="chucker_export_http_confirmation">Export all http calls</string>
```

Comment:
![Screenshot 2020-04-05 at 16 53 45](https://user-images.githubusercontent.com/13467769/78500580-ed984800-775f-11ea-98d3-f7ee6a0fd2f5.png)
This dialog seems not that user friendly and needs some texts updates.

```suggestion
    <string name="chucker_export_http_confirmation">Do you want to export all network transactions?</string>
```

Reply:
updated